doc/src/man/litani-transform-jobs.scdoc

```diff
@@ -0,0 +1,153 @@
+litani-transform-jobs(1) "" "Litani Build System"
```

Comment:
also, add new Litani capability here: https://github.com/awslabs/aws-build-accumulator/blob/release/doc/src/man/litani.scdoc#L87

Reply:
Done, thank you!